script.js

```diff
@@ -480,3 +479,44 @@ function sketch(p) {
     p.rect(x, y, w, h);
   }
 };
+
+if (navigator.mediaSession) {
+
+  function playAudio() {
+    var audio = document.createElement('audio');
+    audio.src = './silence.mp3'; // silence must be at least 5 seconds https://freesound.org/people/silentspeaker/sounds/426895/
```

Comment:
I'm not sure I understand what this is and why it's needed. Do you have a link to a reference or the spec?

Reply:
https://developers.google.com/web/updates/2017/02/media-session#implementation_notes

see point 5. specifically

> As the Web Audio API doesn't request Android Audio Focus for historical reasons, the only way to make it work with the Media Session API is to hook up an <audio> element as the input source to the Web Audio API. Hopefully, the proposed Web AudioFocus API will improve the situation in the near future.

Reply:
Ah, then can you leave that sentence and the link as a comment, please?